openslides_backend/action/mixins/singular_action_mixin.py

```diff
@@ -2,6 +2,7 @@
 
 from ...shared.exceptions import ActionException
 from ...shared.schema import schema_version
+from ..action import Action
 from ..util.typing import ActionPayload
 
 singular_schema = fastjsonschema.compile(
```

Comment:
This schema is obsolet now, you can just use the `action_payload_schema`

Reply:
I do not understand you objection. The SingularActionMixin ensures that one can only send one item inside specific action payloads.

Reply:
Please open new PR if something is still open.